apps/fluent-tester/src/TestComponents/TabList/TabListTest.tsx

```diff
@@ -21,7 +21,7 @@ const PaddedTabList = TabList.customize({
   paddingVertical: 4,
 });
 
-const TabListDefaultTest: React.FunctionComponent = () => {
+export const TabListDefaultTest: React.FunctionComponent = () => {
```

Comment:
>export  [](http://example.com/codeflow?start=0&length=7)

Why is this exported?

Reply:
Originally for importing to the root of the tester to isolate as the only react being rendered in the app. Will be removed.